Description: Remove unaligned mem-accesses from test_malloc_pools.cpp
--- a/src/test/test_malloc_pools.cpp
+++ b/src/test/test_malloc_pools.cpp
@@ -58,11 +58,10 @@
 
 static void *getMallocMem(intptr_t /*pool_id*/, size_t &bytes)
 {
-    void *rawPtr = malloc(bytes+sizeof(MallocPoolHeader)+1);
+    void *rawPtr = malloc(bytes+sizeof(MallocPoolHeader));
     if (!rawPtr)
         return NULL;
-    // +1 to check working with unaligned space
-    void *ret = (void *)((uintptr_t)rawPtr+sizeof(MallocPoolHeader)+1);
+    void *ret = (void *)((uintptr_t)rawPtr+sizeof(MallocPoolHeader));
 
     MallocPoolHeader *hdr = (MallocPoolHeader*)ret-1;
     hdr->rawPtr = rawPtr;
